api/v1beta1/hyperconverged_types.go

```diff
@@ -126,6 +126,11 @@ type HyperConvergedSpec struct {
 	// the value - the higher the log verbosity.
 	// +optional
 	LogVerbosityConfig *LogVerbosityConfiguration `json:"logVerbosityConfig,omitempty"`
+
+	// TektonPipelinesNamespace defines namespace in which example pipelines will
+	// be deployed.
+	// +optional
+	TektonPipelinesNamespace *string `json:"tektonPipelinesNamespace,omitempty"`
```

Comment:
is it really worth to let the user tune this or should HCO simply provide an opinionated fixed value?

Reply:
since we let user to set e.g. common - template namespace, I thought we would like to let user set this namespace as well.